app/models/post.rb

```diff
@@ -1,4 +1,7 @@
 class Post < ApplicationRecord
+  validates :title, presence: true, length: { maximum: 250 }
+  validates :comments_counter, :likes_counter, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
✅ CommentsCounter must be an integer greater than or equal zero 👍🏻 
✅ LikesCounter must be an integer greater than or equal zero 👍🏻